src/pages/periodValidity.tsx

```diff
@@ -99,14 +107,18 @@ const PeriodValidity = ({ errors = [], csrfToken }: PeriodValidityProps): ReactE
 };
 
 export const getServerSideProps = (ctx: NextPageContextWithSession): { props: PeriodValidityProps } => {
+    let errors: ErrorInfo[] = [];
     const csrfToken = getCsrfToken(ctx);
     const periodExpiryAttribute = getSessionAttribute(ctx.req, PERIOD_EXPIRY_ATTRIBUTE);
 
-    if (periodExpiryAttribute && isPeriodExpiryWithErrors(periodExpiryAttribute)) {
-        const { errorMessage } = periodExpiryAttribute;
-        return { props: { errors: [{ errorMessage, id: errorId }], csrfToken } };
+    if (isWithErrors(periodExpiryAttribute)) {
+        errors = periodExpiryAttribute.errors;
+        const fieldsets: RadioConditionalInputFieldset[] = getFieldsets(errors);
+        return { props: { errors, fieldsets, csrfToken } };
```

Comment:
We could remove these two lines, as they will be handled by the two lines below (with a slight change to line 121)?